### PR TITLE
correct unit error in geoaxes.tissot() (m->km)

### PR DESCRIPTION
**Pull request by skjaeve**
The tissot() docstring says the radius input is in km, but the underlying Geodesic.circle() function takes it to be meters. This patch adds a conversion from meters to kilometers.

An alternative solution would be to change the docstring to say meters; this wouldn't break existing code using tissot().

### CODE REVIEW

File: lib/cartopy/mpl/geoaxes.py [hunk 1]
```diff
@@ -418,7 +418,7 @@ def coastlines(self, resolution='110m', color='black', **kwargs):
                                                       resolution, **kwargs)
         return self.add_feature(feature)
 
-    def tissot(self, rad_km=5e5, lons=None, lats=None, n_samples=80, **kwargs):
+    def tissot(self, rad_km=5, lons=None, lats=None, n_samples=80, **kwargs):
```
Comment (QuLogic) [inline lib/cartopy/mpl/geoaxes.py]: The equivalent is 500, I guess, not 5.

**Comment by QuLogic** [comment]
Ping @scmc72.

**Comment by QuLogic** [comment]
Ping @scmc72.

**Comment by QuLogic** [comment]
Ping @skjaeve (have I been pinging the wrong person all this time?)

File: lib/cartopy/mpl/geoaxes.py (hunk 1 above)
Comment (skjaeve) [inline lib/cartopy/mpl/geoaxes.py]: Oops, you're absolutely right.

**Comment by skjaeve** [comment]
Hello. Sorry, I'm not familiar with how pull requests work. Should I change the request?
Would  you prefer that the number is correct, or change the docstring?

**Comment by skjaeve** [comment]
Fixed default argument committed.

**Comment by ajdawson** [comment]
This PR has gone a bit wrong, looks like an accidental merge of master into your branch. @skjaeve - are you able to get this back into shape, we should only be seeing the commits you made for this PR (I think 0d2d518 and 6ddbd92, but please check).

**Comment by ajdawson** [comment]
Also, we'll need you to complete the [CLA](http://scitools.org.uk/documents/cla_form.pdf) before we can merge the PR @skjaeve. For general information on CLAs see http://oss-watch.ac.uk/resources/cla.

**Comment by skjaeve** [comment]
Yup, I bungled it. I didn't realise merging changes from the upstream branch would also mess up the pull request (I thought commits already in the upstream code would be filtered out, since there's nothing to merge.) The simplest solution is probably to withdraw this pull request and make another one.

**Comment by skjaeve** [comment]
See request #904 instead.

**Pull request closed without merging**